backend/tests/unit/apps/slack/commands/report_test.py

```
@@ -0,0 +1,211 @@
+from apps.slack.commands.report import Report
+from apps.slack.enums import ReportSource
+from apps.slack.utils.report_modal import (
+    FEATURE_OFF_TEXT,
+    INVALID_LINK_TEXT,
+    NOT_VISIBLE_TEXT,
+    USAGE_TEXT,
+    decode_metadata,
+)
+
+
+def enabled_workspace(**kwargs):
```

Ruff (0.16.1)

[warning] Missing type annotation for `**kwargs`

(ANN003)